### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by vvbandeira**
We are unifying the documentation of ORFS in the readthedocs.
This PR should be considered (and merged) after [this PR](https://github.com/The-OpenROAD-Project/OpenROAD/pull/932) from the OpenROAD repo.

### CODE REVIEW

**Comment by rovinski** [comment]
I am thoroughly confused about this. ORFS is separate from OpenROAD. Where is the readthedocs source coming from? Is the OpenROAD repo going to have documentation on ORFS or vice versa?

This PR deletes a info about the flow that I don't see replicated in the other PR. Is this just going away?

**Comment by vvbandeira** [comment]
> I am thoroughly confused about this. ORFS is separate from OpenROAD. Where is the readthedocs source coming from? Is the OpenROAD repo going to have documentation on ORFS or vice versa?

@rovinski we are trying to centralize the documentation to avoid having duplicated text, instructions and guidelines. The goal is to have the project documentation (OR + ORFS) integrated into readthedocs. Indeed that means that the source code for the readthedocs will live in the OR repo, but the goal of keeping everything centralized is to reduce the work to maintain documentation.

> This PR deletes a info about the flow that I don't see replicated in the other PR. Is this just going away?

The goal was to move and only remove things that are no longer relevant. Could you point to the particular info? I will double check and add if necessary.

**Comment by rovinski** [comment]
Ok that could be a bit confusing, but I can see how that makes sense. I would advise structuring the readthedocs so that OR and ORFS usage instructions are completely separate. Ideally it would probably be broken into 3 main sections: 1) Users who just want to run their design through ORFS with minimal effort 2) Experienced users who want to change ORFS and use the OR API 3) Developers who want to implement new features in OR.

> Could you point to the particular info?

I'm mainly looking at the flow readme.md that got deleted.

File: README.md [hunk 1]
```diff
@@ -1,150 +1,16 @@
...
-zero loss of power-performance-area (PPA) design quality. No humans means that
-tools must adapt and self-tune, and never get stuck: thus, machine intelligence
-must replace today's human intelligence within the layout generation process. 24
-hours means that problems must be aggressively decomposed into bite-sized
-subproblems for the design process to remain within the schedule constraint.
-Eventual zero loss of PPA quality requires parallel and distributed search to
-recoup the solution quality lost by problem decomposition.
+See the OpenROAD [documentation here](https://openroad.readthedocs.io/en/latest/).
+How to [start using OpenROAD flow here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html)
+and our [user guide here](https://openroad.readthedocs.io/en/latest/user/UserGuide.html).
```
Comment (rovinski) [inline README.md]: Can this be a bullet list? It would be nice to have each of these items on a separate line.

**Review by rovinski — APPROVED** [review]
Looks ok as long as there are plans to bring the flow documentation (especially documentation about the platforms repo) into the readthedocs.

**Comment by vvbandeira** [comment]
> Ok that could be a bit confusing, but I can see how that makes sense. I would advise structuring the readthedocs so that OR and ORFS usage instructions are completely separate. Ideally it would probably be broken into 3 main sections: 1) Users who just want to run their design through ORFS with minimal effort 2) Experienced users who want to change ORFS and use the OR API 3) Developers who want to implement new features in OR.

Yes, we are trying to keep them well separated. If you look at [this temporary webpage ](https://openroad-test.readthedocs.io/en/latest/) that includes the proposed changes. On the left side bar we are keeping OR and ORFS separated, but at the same time we include the Getting Involved, Developer Guide, FAQs outside them as they tend to be related to the "big picture". As always, suggestions are more than welcome.

> > Could you point to the particular info?
> 
> I'm mainly looking at the flow readme.md that got deleted.

I see, for reference the new place for the main README and flow/README sections:

- [Code Organization](https://github.com/vvbandeira/OpenROAD/blob/new-docs/docs/user/UserGuide.md#code-organization)
- [How to build](https://github.com/vvbandeira/OpenROAD/blob/new-docs/docs/user/GettingStarted.md#setup)
- [Contributing](https://github.com/vvbandeira/OpenROAD/blob/new-docs/docs/index.md#how-to-contribute)
- [About](https://github.com/vvbandeira/OpenROAD/blob/new-docs/docs/index.md#welcome-to-openroads-documentation)
- [How to run, design / platform organization](https://github.com/vvbandeira/OpenROAD/blob/new-docs/docs/user/GettingStarted.md)

File: README.md (hunk 1 above)
Comment (vvbandeira) [reply README.md]: Fixed on 697aad5

**Comment by rovinski** [comment]
> If you look at [this temporary webpage ](https://openroad-test.readthedocs.io/en/latest/) that includes the proposed changes.

I see now. Looks fantastic 👍

**Pull request merged**